Source/WebKit/Resources/SandboxProfiles/ios/com.apple.WebKit.GPU.sb.in

```diff
@@ -93,6 +93,32 @@
            (global-name "com.apple.coremedia.visualcontext.xpc")
            (global-name "com.apple.coremedia.volumecontroller.xpc"))
 
+#if HAVE(MEDIAPLAYBACKD)
+    (allow mach-lookup
+        (global-name
+            "com.apple.coremedia.mediaplaybackd.asset.xpc"
+            "com.apple.coremedia.mediaplaybackd.assetimagegenerator.xpc"
+            "com.apple.coremedia.mediaplaybackd.audiodeviceclock.xpc"
+            "com.apple.coremedia.mediaplaybackd.audioprocessingtap.xpc"
+            "com.apple.coremedia.mediaplaybackd.cpe.xpc"
+            "com.apple.coremedia.mediaplaybackd.cpeprotector.xpc"
+            "com.apple.coremedia.mediaplaybackd.customurlloader.xpc"
+            "com.apple.coremedia.mediaplaybackd.figcontentkeysession.xpc"
+            "com.apple.coremedia.mediaplaybackd.figcpecryptor.xpc"
+            "com.apple.coremedia.mediaplaybackd.formatreader.xpc"
+            "com.apple.coremedia.mediaplaybackd.player.xpc"
+            "com.apple.coremedia.mediaplaybackd.remaker.xpc"
+            "com.apple.coremedia.mediaplaybackd.samplebufferaudiorenderer.xpc"
```

Comment:
Are there other services we can remove access to when mediaaybackd is present? I may look like there are old counterparts to some of these new services?

Reply:
Perhaps we can add telemetry to the old services if HAVE_MEDIAPLAYBACKD = 1?

Reply:
@pvollan: Indeed, we should consider that in the future, but for now the relevant people working on this finer-grained XPC layer say they do not intend for us to remove the old items yet (though this will hopefully happen soon).